src/modules/referral/services/queries.ts

```diff
@@ -94,7 +94,6 @@ export const updateStickyReferralAddresses = () => {
       inner join deposit d4
         on d3."depositorAddr" = d4."depositorAddr" and d3."depositDate" >= d4."depositDate" and 
           d4."referralAddress" is not null
-      where d3."depositorAddr" = $1
```

Comment:
update all sticky referral addresses, not only for a particular address. This query will run inside a cron job every 5 minutes